_data/team/roles/non_core_library_maintenance.yml

```diff
@@ -0,0 +1,80 @@
+role: Non-core library maintenance
+description: |
+  The non-core library maintenance team is responsible for the
+  managing the maintenance of the all MDAnalysis packages
+  that are not the core MDAnalysis library.
```

Comment:
Since they're under the MDA organization, perhaps they just need adding as separate sub-roles? @IAlibay also suggested combining all these non-core packages under one general non-core maintenance team instead of itemizing each library, which seemed like a great idea -- I hadn't previously implement this as I was trying to avoid changing things from the Tasks&Roles spreadsheet as much as possible, but if we do so we could organize as follows:

* individual MDAKits with their individual leads
* all other non-core (pyedr, cookiecutter, etc)

Alternatively, we could make MDAKits their own role too where each kit is a subrole.